packages/block-library/src/media-text/edit.js

```diff
@@ -267,86 +272,136 @@ function MediaTextEdit( {
 	};
 
 	const mediaTextGeneralSettings = (
-		<PanelBody title={ __( 'Settings' ) }>
-			<RangeControl
-				__nextHasNoMarginBottom
-				__next40pxDefaultSize
```

Comment:
Isn't it better to leave `__next40pxDefaultSize` as is?

Reply:
Yes, this is meant to be untouched! thank you.